src/acceptance/app/specificdate_schedule_policy_test.go

```diff
@@ -0,0 +1,107 @@
+package app
+
+import (
+	"acceptance/config"
+	"fmt"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var _ = Describe("AutoScaler specific date schedule policy", func() {
+	var (
+		appName              string
+		appGUID              string
+		instanceName         string
+		initialInstanceCount int
+		location             *time.Location
+		endDateTime          time.Time
+	)
+
+	BeforeEach(func() {
+		instanceName = generator.PrefixedRandomName("autoscaler", "service")
+		createService := cf.Cf("create-service", cfg.ServiceName, cfg.ServicePlan, instanceName).Wait(cfg.DefaultTimeoutDuration())
+		Expect(createService).To(Exit(0), "failed creating service")
+	})
+
+	JustBeforeEach(func() {
+		appName = generator.PrefixedRandomName("autoscaler", "nodeapp")
+		countStr := strconv.Itoa(initialInstanceCount)
+		createApp := cf.Cf("push", appName, "--no-start", "-i", countStr, "-b", cfg.NodejsBuildpackName, "-m", cfg.NodeMemoryLimit, "-p", config.NODE_APP, "-d", cfg.AppsDomain).Wait(cfg.DefaultTimeoutDuration())
+		Expect(createApp).To(Exit(0), "failed creating app")
+
+		guid := cf.Cf("app", appName, "--guid").Wait(cfg.DefaultTimeout)
+		Expect(guid).To(Exit(0))
+		appGUID = strings.TrimSpace(string(guid.Out.Contents()))
+
+		Expect(cf.Cf("start", appName).Wait(cfg.DefaultTimeout * 2)).To(Exit(0))
+		waitForNInstancesRunning(appGUID, initialInstanceCount, cfg.DefaultTimeoutDuration())
+	})
+
+	AfterEach(func() {
+		deleteService := cf.Cf("delete-service", instanceName, "-f").Wait(cfg.DefaultTimeoutDuration())
```

Comment:
The order is technically wrong since it should be the reverse of how we create, but it has no real downside.